### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mzouros**
I summon you spyroukostas

### CODE REVIEW

File: eic-registry-rest/src/main/java/eu/einfracentral/registry/controller/UserEventsController.java [hunk 1]
```diff
@@ -84,7 +84,7 @@ public ResponseEntity<List<RichService>> ratings(Authentication auth) {
             serviceRatings.putIfAbsent(userEvent.getService(), userEvent.getValue());
         }
         for (Map.Entry<String, Float> serviceRating : serviceRatings.entrySet()) {
-            services.add(infraServiceService.getRichService(serviceRating.getKey(), "latest", auth));
+            services.add(infraServiceService.getRichService(serviceRating.getKey(), "latest", "eosc", auth));
```
Comment (spyroukostas) [inline eic-registry-rest/src/main/java/eu/einfracentral/registry/controller/UserEventsController.java:87]: same here

File: eic-registry/src/main/java/eu/einfracentral/registry/manager/PendingServiceManager.java [hunk 2]
```diff
@@ -114,7 +123,7 @@ public InfraService transformToPending(InfraService infraService, Authentication
     public InfraService transformToPending(String serviceId, Authentication auth) {
         logger.trace("User '{}' is attempting to transform the Active Service with id {} to Pending", auth, serviceId);
         InfraService infraService = infraServiceService.get(serviceId);
-        Resource resource = infraServiceService.getResource(infraService.getService().getId(), infraService.getService().getVersion());
+        Resource resource = infraServiceService.getResource(infraService.getService().getId(), "eosc", infraService.getService().getVersion());
```
Comment (spyroukostas) [inline eic-registry/src/main/java/eu/einfracentral/registry/manager/PendingServiceManager.java:126]: property here as well

File: eic-registry-rest/src/main/java/eu/einfracentral/registry/controller/UserEventsController.java [hunk 3]
```diff
@@ -62,7 +62,7 @@ public ResponseEntity<List<RichService>> favourites(Authentication auth) {
         }
         for (Map.Entry<String, Float> favouriteService : favouriteServices.entrySet()) {
             if (favouriteService.getValue() == 1) { // "1" is true
-                services.add(infraServiceService.getRichService(favouriteService.getKey(), "latest", auth));
+                services.add(infraServiceService.getRichService(favouriteService.getKey(), "latest", "eosc", auth));
```
[the remark this answers is not in the document]
Comment (mzouros) [reply eic-registry-rest/src/main/java/eu/einfracentral/registry/controller/UserEventsController.java]: done

File: eic-registry/src/main/java/eu/einfracentral/registry/manager/PendingServiceManager.java (hunk 2 above)
Comment (mzouros) [reply eic-registry/src/main/java/eu/einfracentral/registry/manager/PendingServiceManager.java]: done

File: eic-registry-rest/src/main/java/eu/einfracentral/registry/controller/UserEventsController.java (hunk 1 above)
Comment (mzouros) [reply eic-registry-rest/src/main/java/eu/einfracentral/registry/controller/UserEventsController.java]: done

File: eic-registry-rest/src/main/java/eu/einfracentral/registry/controller/ServiceController.java [hunk 4]
```diff
@@ -66,7 +66,7 @@ public ResponseEntity<InfraService> delete(@PathVariable("id") String id, @PathV
             service = infraService.get(id, catalogueId);
         }
         // Block users of deleting Services of another Catalogue
-        if (!service.getService().getCatalogueId().equals("eosc")){
+        if (!service.getService().getCatalogueId().equals(catalogueId)){
```
Comment (spyroukostas) [inline eic-registry-rest/src/main/java/eu/einfracentral/registry/controller/ServiceController.java:69]: fix this

**Review by spyroukostas — APPROVED** [review]

**Pull request merged**